draft-ietf-tls-tls13.md

```diff
@@ -3159,37 +3165,41 @@ The exact source of each of these secrets depends on the operational
 mode (DHE, ECDHE, PSK, etc.) and is summarized in the table below:
 
 ~~~
-    Key Exchange            Static Secret (SS)    Ephemeral Secret (ES)
-    ------------            ------------------    ---------------------
-    (EC)DHE                   Client ephemeral         Client ephemeral
-    (full handshake)       w/ server ephemeral      w/ server ephemeral
+    Key Exchange     Static Secret (SS)    Ephemeral Secret (ES)
+    ------------     ------------------    ---------------------
+    1-RTT (EC)DHE       N/A (SS = ES)        Client Ephemeral
+                                           + Server Ephemeral
```

Comment:
Yes, that's what I said in the previous comment. They're generated from the same values, but the extracted versions are different. This table here is specifically about the sources of these secrets.

My point here is that the notation & terminology should be more clear and spelled out, as evident by our miscommunications here. ;)

The very short abbreviations may be doing more harm than good. Maybe name SS & ES more explicitly; something like raw_ephemeral_secret?